### ISSUE TITLE
wasmtime cranelift log pollutes the containerd log

### ISSUE THREAD

**Issue by Mossaka** [containerd/runwasi#565]
## Reproduce

Run containerd locally by running `make` and then `./bin/containerd --log-level debug` in root user. Then invokes the wasmtime shim to execute the demo wasm container: `sudo ctr run --rm --runtime=io.containerd.wasmtime.v1 ghcr.io/containerd/runwasi/wasi-demo-app:latest testwasm`, then you will see something like:

![image](https://github.com/containerd/runwasi/assets/5447827/e2d4f375-aef9-4b9d-a676-578783d41be1)

## Expected 

We should by default turn off cranelift logs as these are low-level compiler logs which don't fit into operational logs like containerd.

**Comment by z63d** [comment]
We can control the log by changing it as follows, but is this the correct way to do it?
It affects more than just cranelift.

```
log = { workspace = true, features = ["max_level_off", "release_max_level_off"] }
```
https://github.com/containerd/runwasi/blob/775d2b5ff3b9f211ce7ecf25f5bbd5aa86b73be4/crates/containerd-shim-wasmtime/Cargo.toml#L10
https://github.com/bytecodealliance/wasmtime/tree/v31.0.0/cranelift#building-cranelift

**Comment by Mossaka** [comment]
Yeah you are right that this might affect more than just cranelift. What I am looking for is a solution to filter out cranelift logs instead of setting a max_level_off at a global level.
